terraform/modules/tf_stream_alert_kinesis_streams/iam.tf

```diff
@@ -0,0 +1,57 @@
+// IAM User: stream_alert user for systems to send data to the stream
+resource "aws_iam_user" "stream_alert" {
+  count = "${var.create_user ? 1 : 0}"
+  name  = "${var.prefix}_${var.cluster_name}_stream_alert_user"
+  path  = "/streamalert/"
+}
+
+// IAM Group: stream_alert clustered group
+resource "aws_iam_group" "stream_alert" {
+  count = "${var.create_user ? 1 : 0}"
+  name  = "${var.prefix}_${var.cluster_name}_stream_alert_users"
+  path  = "/"
+}
+
+// IAM Group Membership: Assign stream_alert user to group
+resource "aws_iam_group_membership" "stream_alert" {
+  count = "${var.create_user ? 1 : 0}"
+  name  = "stream-alert-kinesis-user-membership"
+
+  users = [
+    "${aws_iam_user.stream_alert.name}",
+  ]
+
+  group = "${aws_iam_group.stream_alert.name}"
+}
+
+// IAM Group Policy: Allow users in the group to PutRecords to Kinesis
+resource "aws_iam_group_policy" "stream_alert_kinesis_put_records" {
+  count = "${var.create_user ? 1 : 0}"
+  name  = "KinesisPutRecords"
+  group = "${aws_iam_group.stream_alert.id}"
+
```

Comment:
I don't think there should be a blank line here, did you `terraform fmt`?

Reply:
I guess the Atom plugin didn't catch this one, I think it was originally a docstring and that's why